src/components/ScriptExplorer/styles.module.scss

```diff
@@ -25,3 +25,13 @@
 .outputsFormLabel {
   color: rgba(0, 0, 0, 0.54);
 }
+
+.maxSpend {
+  color: grey;
+  font-size: .7rem!important;
```

Comment:
:eyes:

Reply:
Is there any other way? :grimacing:

Reply:
I blame MUI

Reply:
Not pretty but I just moved this particular style inline and that properly overrides. Part of the problem is that we don't have a consistent approach to styling. This module seems to be using sass. Others have the makeStyles hook.

Reply:
Gotcha, feel free to resolve this.